userguide/one_shot_object_detection/README.md

```diff
@@ -0,0 +1,120 @@
+# One-Shot Object Detection
+
+One-Shot object detection is the task of learning to detect from as few as one example per category. Unlike the Object Detector which requires many varied examples of objects in the real world, the One-Shot detector only requires a single canonical example of an object to train the classifier on and performs predictions for that category in the wild. It is best suited for 2D objects or when there isn't much expected variance in the examples, such as playing cards, logos, road signs and clapperboards. One-Shot detector is not suitable for 3D objects such as faces, animals and cars. 
```

Comment:
- One-Shot object detection is the task of **detecting an object** from as **little** as one example per category. (Not sure what is syntactically correct w.r.t. few vs. little)

- "It is best suited for 2D objects..." Avoid using the word, "it"

- Oxford comma. "cards, logos, road signs and clapperboards" -> "cards, logos, road signs, and clapperboards" and "faces, animals and cars" -> "faces, animals, and cars"

Reply:
* 'Little' denotes size and 'few' denotes quantity so 'few' is grammatically accurate here.

* I'm not sure I understand why "it" isn't a kosher placeholder? 

* Added Oxford commas.

Reply:
- From https://www.gingersoftware.com/content/grammar-rules/adjectives/few-vs-little/ looks like "little" is used with non-countable nouns and with the singular form (which is true here) whereas "few" is used with countable nouns. Not sure how reliable that link is, but that seems right to me. Feel free to take a judgment call on this.

- To me, "it" makes the sentence confusing especially when the sentence starts with "It". I would recommend replacing "it" with "The detector" but this usage may not be as confusing to everyone.

Reply:
Images are countable hence "few" is the grammatically correct choice.